admin/app/indexes/ContentApiTagsEnumerator.scala

```diff
@@ -39,7 +37,7 @@ object ContentApiTagsEnumerator extends Logging {
   }
 
   def enumerateTagType(tagType: String) = enumeratePages { page =>
-    getResponse(ContentApiClient.tags.tagType(tagType).pageSize(MaxPageSize).page(page))
+    contentApiClient.getResponse(contentApiClient.tags.tagType(tagType).pageSize(MaxPageSize).page(page))
```

Comment:
Oh I see, yes that would make sense to change it later, or at least to split it in two lines :
```scala
val query = ContentApiQuery()
  .tags.tagType(tagType)
  .pageSize(MaxPageSize)
  .page(page)
contentApiClient.execute(query)
```

but as you said that's unrelated